qcelemental/tests/test_models_results.py

```diff
@@ -86,22 +91,84 @@ def test_basis_electron_center_raises():
 
 def test_basis_ecp_center_raises():
     # Check coefficients
-    data = basis_data["bs_def2tzvp_zr"]["ecp_potentials"][0].copy()
+    data = center_data["bs_def2tzvp_zr"]["ecp_potentials"][0].copy()
     data["coefficients"] = [[5, 3]]
 
     with pytest.raises(ValueError):
         basis.ECPPotential(**data)
 
     # Check gaussian_exponents
-    data = basis_data["bs_def2tzvp_zr"]["ecp_potentials"][0].copy()
+    data = center_data["bs_def2tzvp_zr"]["ecp_potentials"][0].copy()
     data["gaussian_exponents"] = [5, 3]
 
     with pytest.raises(ValueError):
         basis.ECPPotential(**data)
 
+
 def test_basis_map_raises():
 
     with pytest.raises(ValueError) as e:
-        assert basis.BasisSet(basis_name="custom_basis",
-                          basis_data=basis_data,
-                          basis_atom_map=["something_odd"])
+        assert basis.BasisSet(name="custom_basis", center_data=center_data, atom_map=["something_odd"])
+
+
+@pytest.fixture(scope="function")
+def wavefunction_data_fixture():
+    bas = basis.BasisSet(name="custom_basis",
+                         center_data=center_data,
+                         atom_map=["bs_sto3g_o", "bs_sto3g_h", "bs_sto3g_h"])
+    c_matrix = np.random.rand(bas.nbf, bas.nbf)
+    mol = qcel.models.Molecule.from_data("""
+        O 0 0 0
+        H 0 0 2
+        H 0 2 0
+    """)
+
+    return {
+        "molecule": mol,
+        "driver": "energy",
+        "model": {
+            "method": "UFF"
+        },
+        "return_result": 5,
+        "wavefunction": {
+            "basis": bas,
+            "scf_orbitals_a": c_matrix,
+            "orbitals_a": "scf_orbitals_a"
```

Comment:
for restricted closed-shell results, should it be convention for beta components to always be reported in addition to alpha ones, especially for return results? This would match the way we handle `calcinfo_nalpha` and `calcinfo_nbeta`.

Reply:
I think we will have to leave it sparse, a factor of two in common cases is hard to give up for these large quantities.

Reply:
I think I'm okay with this, but we should document it somewhere (maybe in the protocols).